tests/integration/records/views/test_views_literature.py

```diff
@@ -205,7 +205,7 @@ def test_literature_citations_with_superseded_citing_records(
 
     expected_status_code = 200
 
-    expected_count = 2
```

Comment:
:cry:

Reply:
the correct is 1 right?

Reply:
yes, it's sad that someone wrote a test for this, but nobody noticed that the test was wrong.